### ISSUE TITLE
[Persistence service] Comment more on availability in README

### ISSUE THREAD

**Issue by jamesscottbrown** [ove/ove-services#101]
The README for the Persistence Service says: "This is also the most straightforward persistence service implementation and is not highly available."

This is potentially worrying to a user: it would be helpful to say something about whether being 'not highly available' is likely to cause problems, and what higher-availability alternatives are available (redis?).

**Comment by senakafdo** [comment]
Yes, I was planning to implement another persistence service(s) that depends on Redis or Mongo. At the moment, I cannot elaborate further since those alternatives are not yet implemented.
